customize.dist/translations/messages.es.js

```diff
@@ -397,9 +397,9 @@ define(function () {
     out.history_closeTitle = "Cerrar el historial";
     out.history_restore = "Restaurar";
     out.history_restoreTitle = "Restaurar la versión seleccionada del documento";
-    out.history_restorePrompt = "¿Estás seguro que quieres cambiar la versión actual del documento por esta?";
+    out.history_restorePrompt = "¿Estás seguro de que quieres cambiar la versión actual del documento por ésta?";
```

Comment:
If you want to follow the new rules by RAE (applicable to `es_ES`, at least), "ésta" should be reverted back to "esta".

Source: http://www.rae.es/consultas/el-adverbio-solo-y-los-pronombres-demostrativos-sin-tilde